lane.py

```diff
@@ -139,13 +192,16 @@ def __init__(self, library, collection, availability, order,
         self.order_ascending = order_ascending
         self.facets_enabled_at_init = enabled_facets
 
-    def navigate(self, collection=None, availability=None, order=None):
+    def navigate(self, collection=None, availability=None, order=None,
+                 entrypoint=None):
```

Comment:
This appears to provide a way to navigate between entry points which contradicts the class's docstring.